arm/Microsoft.Storage/storageAccounts/tableServices/deploy.bicep

```diff
@@ -18,27 +18,28 @@ module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
 
 resource storageAccount 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
   name: storageAccountName
+}
 
-  resource tableServices 'tableServices@2021-04-01' = {
-    name: name
-    properties: {}
-  }
+resource tableServices 'Microsoft.Storage/storageAccounts/tableServices@2021-04-01' = {
+  name: name
+  parent: storageAccount
+  properties: {}
 }
 
 module tableServices_tables 'tables/deploy.bicep' = [for (tableName, index) in tables: {
   name: '${deployment().name}-Storage-Table-${index}'
```

Comment:
'${uniqueString(deployment().name, location)}-storage-table-${index}' ?

Reply:
Why? :D Did we not agree to just use the unique string on the top-level?